docs/ReleaseNotes.md

```diff
@@ -1,6 +1,24 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 3.1
+Acuminator 3.1 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 3.1, the following enhancements have been implemented:
+ - The [PX1056](diagnostics/PX1056.md) diagnostic has been added to check and warn if a graph is created in the `IsActive` method in graph and DAC extension and the `IsActiveForGraph` method.
+ - Code snippets are now provided for generic event handlers and DAC fields. 
```

Comment:
Imho,  we should add a bit more details about code snippets. Maybe create a feature section and add it there. We can at least add a reference to their repository which contains documentation:
 https://github.com/Acumatica/CodeSnippets
 
 Also this is not a full list, it should be like this:
 `Visual Studio Code Snippets are now provided for DAC, DAC fields and graph events.`
 There are templates for DAC and for old graph events with non-generic signatures.